scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java

```diff
@@ -0,0 +1,23 @@
+
+package com.sap.scimono.api.helper;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.sap.scimono.api.API;
+import com.sap.scimono.entity.ErrorResponse;
+
+
+public class JsonMappingExceptionMapper implements ExceptionMapper<JsonMappingException> {
```

Comment:
Check if it is possible to combine this exception mappers into one class

Reply:
It is not possible to make one class implement both ExceptionMapper<JsonMappingException> and ExceptionMapper<JsonParseException>.
Can probably make one class implement ExceptionMapper<JsonProcessingException>

Reply:
No need to add multiple interfaces to achieve this behaviour. The current way is acceptable.